minecraft_servers/phoenix/manifest.json

```diff
@@ -0,0 +1,36 @@
+{
+  "server_name": "phoenix",
+  "nice_name": "Phoenix",
+  "direct_ip": "rainbowfurry.com",
+  "supported_languages": [
+    "de",
+    "en"
+  ],
+  "social": {
+    "web": "https://rainbowfurry.com",
+    "web_shop": "https://phoenix-network-store.tebex.io"
+  },
+  "gamemodes": {
+    "clashofclans": {
+      "name": "Clash of Clans",
+      "command": "/quickjoin CoM",
+      "url": "https://www.rainbowfurry.com/ClashOfClans",
+      "color": "#0095B0",
+      "versions": "1.8.9 - 1.21.3*"
+    }
+  },
+  "discord": {
+    "server_id": 1309574664141668393,
+    "rename_to_minecraft_name": false
+  },
+  "brand": {
+    "primary": "#497cb1",
+    "background": "#497cb1",
+    "text": "#ffffff"
+  },
+  "chat": {
+    "message_formats": [
+      "^§[a-f0-9](?<level>\\d+)( \\||§8 \\|) §[a-f0-9](?<sender>[a-zA-Z0-9_]{2,16})§r§7: §f(?<message>.*)$"
```

Comment:
That's the template message format. Are you sure that you use the same one?

Reply:
I changed it a bit but yes nearly